### PR TITLE
fix master_authorized_networks block not picked up by engine

### PR DESCRIPTION
**Pull request by xingao267**

### CODE REVIEW

File: templates/tfengine/components/resources/gke_clusters/main.tf [hunk 1]
```diff
@@ -29,8 +29,17 @@ module "{{resourceName . "name"}}" {
   ip_range_services      = "{{.ip_range_services_name}}"
   master_ipv4_cidr_block = "{{.master_ipv4_cidr_block}}"
 
-  # Optional.
-  master_authorized_networks = var.master_authorized_networks
+  {{- if has . "master_authorized_networks"}}
+  master_authorized_networks = [
+    {{- range .master_authorized_networks}}
```
Comment (MartinPetkov) [inline templates/tfengine/components/resources/gke_clusters/main.tf:34]: Can you use any of the hcl/hclField functions here? Seems like a list of objects should expand correctly.

Comment (MartinPetkov) [reply templates/tfengine/components/resources/gke_clusters/main.tf:34]: I just tried it. You can do
```
master_authorized_networks = {{hcl .master_authorized_networks}}
```

And the generated files are the same.

Comment (umairidris) [reply templates/tfengine/components/resources/gke_clusters/main.tf:34]: +1 I think you should be able to do {{hclField . "master_authorized_networks" false}}

Comment (xingao267) [reply templates/tfengine/components/resources/gke_clusters/main.tf]: Done!

**Pull request merged**